docs/site/repl-window.md

```diff
@@ -0,0 +1,151 @@
+   [:p "(Check the js console for hints if nothing exciting happens.)"]])
+```
+
+If the REPL Window is set to be the output destination for evaluation results, the REPL window will print the defined var (otherwixe it will be printed to the destination configured). The REPL window will regardless print a new prompt reflecting the current REPL connection and namespace:
+
+![eval-results-1](images/howto/output/eval-results-1.png)
+
+If you then switch to the output window (`ctrl+alt+o r`), and enter this at the prompt:
+
+```clojure
+mount-target
+```
+
+then evaluate it using `alt+enter`, you'll get:
+
+![eval-results-2](images/howto/output/eval-results-2.png)
+
+This, since the namespace ”followed” the first evaluation over to the output window.
+
+## REPL History
+
+Recently evaluated forms in the REPL file are persisted and can easily be shown again for modifying and re-evaluating.
+
+### Navigate REPL History
+
+You can navigate up and down the last forms evaluated in the REPL file by using `alt+up` and `alt+down`, provided your cursor is at the end of the last form after the prompt. If the cursor is not at the end of the last form, then `alt+up` and `alt+down` will do what they are mapped to, which is by default "Move Line Up" and "Move Line Down," respectively.
+
+If you have typed some text after the prompt before you start traversing up the history, this text will be preserved and will display when you traverse back down the history. If you modify some text in the history while traversing, the modification will be saved at that location in history.
+
+### Clear REPL History
+
+You can clear the repl history by running the command "Clear REPL History" from the command palette.
+
+## Stack Traces
+
+When an evaluation produces an error, the output window will automatically print the the error message. If there is a stack trace associated with the error, this can now be printed on demand using the **Calva: Print Last Stacktrace to the Output Window** command. The output window will also have a Codelense button below the error message that will print the stack trace..
+
+![Print Stacktrace Codelense button](images/howto/output/print-stacktrace-codelense.png "Print Stacktrace Codelense button")
+
+For printed stacktraces, when source locations are available (Clojure files) you will be able to navigate to them by pressing `ctrl+click` (`cmd+click` on Mac) on the file name. You can also hover over symbols in the stack trace to see the symbol's documentation, and `ctrl+click` (`cmd+click` on Mac) the symbol to Peek Definition.
+
+![Stack trace clicking and peeking definition](images/howto/output/stack-traces.gif "Stack trace clicking and peeking definition")
+
+!!! Note "Output destinations"
+    If you have configured some other destination for stderr output, the error message will be printed there as well. But it will _also_ be printed to the REPL Window, because the augmented stack traces need this (because reasons). 
```

Comment:
```suggestion
!!! Note "Output destinations"
    Even if you have configured some other destination for stderr output, the error message will be printed there _in addition to_ the REPL Window, as the augmented stack traces need this (because reasons). 
```

Reply:
I'll keep the original here, because it better relays the decision/what's going on. (It's being printed in the REPL window as an additional destination.)